beacon-chain/rpc/eth/beacon/handlers.go

```diff
@@ -2070,3 +2070,43 @@ func (s *Server) GetGenesis(w http.ResponseWriter, r *http.Request) {
 	}
 	httputil.WriteJson(w, resp)
 }
+
+// GetDepositSnapshot returns the deposit snapshot.
```

Comment:
The doc would be more informative if we said what the spec does:
> // Retrieve EIP-4881 Deposit Tree Snapshot. Depending on Accept header it can be returned either as json or as bytes serialzed by SSZ.